presto-product-tests/conf/presto/etc/catalog/mysql.properties

```diff
@@ -0,0 +1,4 @@
+connector.name=mysql
+connection-url=jdbc:mysql://mysql:3306/test
```

Comment:
Docker magically adds all service names to it's virtual DNS, so it is not necessary to edit /etc/hosts.